src/client/components/Timer/Timer.js

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import './Timer.styles.css';
+
+const formatTime = (timeLeftInSecond) => {
+  let minute = Math.floor(timeLeftInSecond / 60);
+  if (minute < 10) minute = `0${minute}`;
+
+  let second = timeLeftInSecond - 60 * minute;
+  if (second < 10) second = `0${second}`;
+
+  return `${minute}:${second}`;
+};
```

Comment:
Nice that you extracted this into a separate function 👏 

A few comments to consider: 
- Avoid using mutations
- There is a function in JavaScript that lets you add the `0` without adding conditions
- Declare all constants you are using e.g. think if someone didn't know what does `60` mean? 

I gave an example of doing the same thing applying the comments above

```javascript

// prepends `0` to single digit number
const getTimeWithDoubleDigits = (numberString) => `${numberString}`.padStart(2,'0');

function formatTime (timeLeftInSecond) {
  const secondsPrMinute = 60;
  // Deriving minutes from total seconds and rounding it
  const minutes = Math.floor(timeLeftInSecond / secondsPrMinute);
  const formattedMinutes = getTimeWithDoubleDigits(minute);

 //  Deriving remaining seconds from total seconds
  const seconds = timeLeftInSecond - secondsPrMinute * minutes;
  const formattedSeconds = getTimeWithDoubleDigits(seconds);

  return `${formattedMinutes}:${formattedSeconds}`;
};
```

Reply:
An example that it works the same as yours 👍 

![image](https://user-images.githubusercontent.com/16811871/83243135-c8431b80-a19d-11ea-94b0-8552b0a6fc11.png)